projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -333,12 +332,12 @@ private boolean answer(String questionTemplateName, String paramsLine,
     * type.
     * @throws BatfishException If the type encoded in input {@code value}
     * does not match the type specified in {@code variable}.
-    * @throws InvalidStateException If the input {@code value} is null.
+    * @throws IllegalArgumentException If the input {@code value} is null.
```

Comment:
Also, the exception should be thrown in the caller that is extracting `JsonNode` from the parameter map – not here in the `validateType` function.

This function should simply assume that the `value` is non-null and does not need to handle that case or declare an extra exception for the case that it is null.